### ISSUE TITLE
𝕏 (formerly, Twitter) icon not showing in website footer

### ISSUE THREAD

**Issue by ArvindParekh** [kubernetes/website#46405]
**This is a Feature Request**

**What would you like to be added**

The [k8s.io](https://kubernetes.io/) website footer doesn't have a link to its 𝕏 account ([@kubernetesio](https://twitter.com/kubernetesio)), which is now maintained by the Kubernetes community alongside CNCF. I suggest we should add that.

![image](https://github.com/kubernetes/website/assets/71211731/4bf17c51-b47d-452a-afb5-a5692210fb0f)

**Why is this needed**

**Comments**

I guess I should add it to the left group. Let me know if there's any feedback.

**Comment by dipesh-rawat** [comment]
Thank you for reporting this issue! It appears that there is already 𝕏 detail in the footer _(as seen the HTML source below),_ but the icon isn't rendering.


```html
<li class="list-inline-item mx-2 h3" data-toggle="tooltip" data-placement="top" title="" 
aria-label="X(Twitter)" data-original-title="X(Twitter)">
       <a class="text-white" target="_blank" href="https://twitter.com/kubernetesio">
            <i class="fab fa-x-twitter"></i>
        </a>
</li>
```

**Comment by dipesh-rawat** [comment]
This looks like a bug.
/remove-kind feature

/retitle 𝕏 (formerly, Twitter) icon not showing in website footer
/kind bug
/area web-development

**Comment by dipesh-rawat** [comment]
Previously, the old Twitter logo was rendering properly (refer to the [v1.28 docs website (here)](https://v1-28.docs.kubernetes.io/docs/home/)). 

I think it's after our work involved to update to " 𝕏 " that the logo stopped rendering (refer to the commit  https://github.com/kubernetes/website/commit/803bb6b16b6d678fd5b3ecef1debde6e81865081 where it was added). It could be that the version of Font Awesome being used by the current version of Docsy is outdated and doesn't include the new icon.

The footer  𝕏 icon details are pulled from [hugo.toml](https://github.com/kubernetes/website/blob/main/hugo.toml) (refer below).
https://github.com/kubernetes/website/blob/44669d56d7329c0a12eeaac31318f3025a91e51e/hugo.toml#L252-L255

**Comment by dipesh-rawat** [comment]
We should fix this issue in the footer.

/priority backlog
/triage accepted

**Comment by tengqm** [comment]
fontawsome 4.7.0 doesn't include this icon, the rename only happened a short time ago.

**Comment by ArvindParekh** [comment]
Oh got it! Thank you for explaining and providing the links @dipesh-rawat! 

>  Try search "x-twitter" on https://fontawesome.com/v4/icons/.

I did, v4 does not include the X Twitter icon, even v5 doesn't have it, it was introduced in the latest version (v6) of fontAwesome.

> It could be that the version of Font Awesome being used by the current version of Docsy is outdated and doesn't include the new icon.

I just checked the [latest Docsy release](https://github.com/google/docsy/releases/tag/v0.10.0), where they have bumped fontAwesome to the latest release version 6.5.2 (see [#1925](https://github.com/google/docsy/pull/1925))

Which means, updating the icon name in [hugo.toml](https://github.com/kubernetes/website/blob/44669d56d7329c0a12eeaac31318f3025a91e51e/hugo.toml#L252-L255) (to `fa-brands fa-x-twitter`, which are the classnames for x-twitter in the latest release)  should fix this issue, right?

**Comment by Shubham82** [comment]
fontawsome 6.5.2 includes `x-twitter` icon, 
https://fontawesome.com/v6/search?q=twitter&o=r

**Comment by chris-short** [comment]
@sftim and I were discussing this very topic this week. I checked FontAwesome too and couldn't distinguish the differences.

**Comment by ktvargo-ms** [comment]
> > It could be that the version of Font Awesome being used by the current version of Docsy is outdated and doesn't include the new icon.
> 
> I just checked the [latest Docsy release](https://github.com/google/docsy/releases/tag/v0.10.0), where they have bumped fontAwesome to the latest release version 6.5.2 (see [#1925](https://github.com/google/docsy/pull/1925))

Yes, it looks this uses an older version of Docsy and an upgrade to the latest/newer version of Docsy is required to get newer fontawesome fonts.  Of course, that impacts the entire site.

**Comment by chris-short** [comment]
FontAwesome is a submodule of Docsy if I recall. That could be updated independently I would imagine. I do manually adds to my personal site's iconography routinely.

cc @jberkus

**Comment by aakankshabhende** [comment]
@chris-short @ArvindParekh the font awesome icon for X is available now. Can I work on this issue?

**Comment by chris-short** [comment]
You don't need permission. Feel free to fix this.

**Comment by chris-short** [comment]
We discussed this during today's Contributor Comms meeting and more in Slack after the meeting in the comms channel. At this point, we commit to taking X off both sites and have Bluesky, Mastodon, and LinkedIn accounts added to the footers.

We will still post to X for now, as it's our largest audience, but the platform's engagement and our account's reach are declining. We need a more in-depth analysis/discussion before deprecating X altogether.

If you have strong feelings about social media, WE HEAR YOU. Feel free to ping me on Slack (chris-short there, too) if you want to share opinions or ideas. As always, feel free to join the Comms meeting to discuss things with mouthwords.

**Comment by chris-short** [comment]
I will close this as I removed X/Twitter from both sites and those PRs should merge soon.

/close
